engine/render/src/render/render_command.cpp

```diff
@@ -76,12 +76,12 @@ namespace dmRender
                 }
                 case COMMAND_TYPE_SET_RENDER_TARGET:
                 {
-                    dmGraphics::SetRenderTarget(context, (dmGraphics::HRenderTarget)c->m_Operands[0], c->m_Operands[1] );
+                    dmGraphics::SetRenderTarget(context, dmGraphics::MakeAssetHandle(c->m_Operands[0], dmGraphics::ASSET_TYPE_RENDER_TARGET), c->m_Operands[1] );
```

Comment:
I would not have thought that we'd create a new handle for each command.
Is this what we want?

Also, for debugging, this handle will be different from the one sent from elsewhere, which makes it harder to debug?